### ISSUE TITLE
[BUG] 

### ISSUE THREAD

**Issue by GenLangsdorf** [ShammyLevva/FTAnalyzer#173]
**Version Number problem appears in** 7.8.2.0

**Describe the bug**
Once the gedcom is loaded I get an error message

**To Reproduce**
1. Go to '...' Open program
2. Click on '....' Load Gedcom

**Expected behavior**
Should be able to analyse the tree
![Capture](https://user-images.githubusercontent.com/30801191/83965077-f2d15a80-a8a8-11ea-8f59-c57a0e1c80e9.JPG)

**Screenshots**

**Additional context**

**Comment by ShammyLevva** [comment]
Not a bug - problem on user machine that has prevented write access to the folder mentioned. Use the Microsoft Store reset program to try to address issue. Failing that manually set permissions on the folder or use the release version.

**Comment by ShammyLevva** [comment]
Yup sadly it's not a bug with my software it's a bug with the Microsoft Store on your machine. The Microsoft Store installer is failing to create the folder on the PC. 

I've not been able to get to the bottom of why

**Comment by GenLangsdorf** [comment]
I've tried installing from GitHub but I just get to a program that says it no longer works and to use the MS store version.  When I originally installed the Store version, I didn't install on my C drive as I like to keep that free.  Could that be the problem?  I can't find a way to uninstall it and re-install on my C drive either so I'm going round in circles.  
I would be happy to help out, but I'm not a big techie.  I've struggled installing the Github version which seems strangely complex.  Just let me know what you need to do

**Comment by ShammyLevva** [comment]
The GitHub version is at www.ftanalyzer.com/releases

**Comment by GenLangsdorf** [comment]
I think it was an option on install but I could be thinking of something else.  The link you sent works fine and I have the program up and running properly now - installed on the default drive
